deploy/kubernetes/overlays/examples/stackdriver-sidecar.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: apps/v1beta2 # for versions before 1.8.0 use extensions/v1beta1 [apps/v1beta2]
+kind: DaemonSet
+metadata:
+  name: cadvisor
+spec:
+  template:
+    spec:
+      containers:
+      - name: prometheus-to-sd
+        image: gcr.io/google-containers/prometheus-to-sd:v0.2.6
+        ports:
+          - name: profiler
+            containerPort: 6060
```

Comment:
6061? https://github.com/GoogleCloudPlatform/k8s-stackdriver/blob/master/prometheus-to-sd/main.go#L62

Reply:
I was following https://github.com/GoogleCloudPlatform/k8s-stackdriver/blob/master/prometheus-to-sd/kubernetes/prometheus-to-sd-kube-state-metrics.yaml#L30, but maybe they got it wrong

Reply:
yeah, they got it wrong